TYPO3.Flow/Documentation/TheDefinitiveGuide/PartIII/AspectOrientedProgramming.rst

```diff
@@ -971,6 +971,34 @@ The following example introduces a new interface ``NewInterface`` to the class
 		}
 	}
 
+Trait introduction
+-------------------
+
+Like the interface introductions, also trait introductions are declared
+by annotation. It even uses the same annotation with a different argument:
+
+``@Flow\Introduce("PointcutExpression", traitName="NewTraitName")``
+
+Again only pointcuts filtering for classes make sense. The traitName must
+be a fully qualified "class" (trait) name without leading backslash.
+
+The following example introduces a trait ``SomeTrait`` to the class ``MyClass``.
```

Comment:
Mmm, but that is the same for interface introduction. The example says you need to create an (aspect) class.